### ISSUE TITLE
Port out of range error when used in host string

### ISSUE THREAD

**Issue by tbnguyen1407** [alajmo/sake#38]
- [x] I have the latest version of sake
- [x] I have searched through the existing issues

## Info

- OS
  - [x] Linux
  - [ ] Mac OS X
  - [ ] Windows
  - [ ] other

- Shell
  - [x] Bash
  - [ ] Zsh
  - [ ] Fish
  - [ ] Powershell
  - [ ] other

- Version: 0.12.0

## Problem / Steps to reproduce

```
~ sake.yaml ~
servers:
  myserver1:  ## not working (error: strconv.ParseInt: parsing "55555": value out of range)
    host: user@1.2.3.4:55555
  myserver2:  ## working
    host: user@1.2.3.4
    port: 55555
```

```
sake check
```

**Comment by alajmo** [comment]
Acknowledged, will push fix today.

**Comment by alajmo** [comment]
Fixed with v0.12.1
